### PR TITLE
Request to add helpr to the probot listing

### LINKED ISSUE #2: Explore apps
There are currently only 6 apps listed on the homepage, with a button to "explore more apps". This should take you to a page where you can browse all apps by:

- Popularity (installs)
- Topic (TODO: curate initial list of topics)
- ???

### PR DESCRIPTION
**Pull request by rsarky**
I made an app that assigns labels to issues whenever a PR references it like `Fixes #2` :
[helPR](https://github.com/rsarky/helpr)

### CODE REVIEW

File: _apps/helpr.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+</div>
+
+## Usage
+
+Using helpr is pretty straightforward. Just [install](https://github.com/apps/helpr) the app on your repository and you are ready to go. :100: 
+
+
+Consider this open issue :
+
+![issue](./screenshots/usage-img2.jpg)
```
Comment (hiimbex) [inline _apps/helpr.md]: this can't be a relative link on the site

File: _apps/helpr.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+<br><br>
+
+Now when someone references this issue in a pull request helpr will automatically add the label `pr-available` :
+
+![labelled](./screenshots/usage-img1.jpg)
```
Comment (hiimbex) [inline _apps/helpr.md]: same as above

File: _apps/helpr.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+```sh
+# Install dependencies
+npm install
+
+# Run the bot
+npm start
+```
+
+See [docs/deploy.md](docs/deploy.md) if you would like to run your own instance of this app.
```
Comment (hiimbex) [inline _apps/helpr.md]: This setup section probably isn't necessary for the site

**Comment by rsarky** [comment]
> Love the name! 

Thanks. I love puns :smile: 

I have updated the relative links and removed the `setup` section.

**Comment by hiimbex** [comment]
Thanks! Tests are failing related to #171

**Pull request merged**